trie/node.go

```diff
@@ -279,3 +279,26 @@ func shouldStopIfContextDone(ctx context.Context, idleProvider IdleNodeProvider)
 		}
 	}
 }
+
+func getChildForSnapshot(db pruningStorer, childHash []byte) ([]byte, bool, error) {
```

Comment:
misleading: this actually returns the bytes rather than the child. 
Consider renaming this to getChildBytesForSnapshot

Reply:
Renamed to getNodeBytesForSnapshot.